pkg/transponder/configure.go

```diff
@@ -100,9 +99,9 @@ func Configure(cmd *cobra.Command, args []string) error {
 	}
 
 	fmt.Println("Initiating database connection")
-	fmt.Println("Delivering blueberries and pancakes")
 
 	if resp.StatusCode == http.StatusNoContent {
+		fmt.Println("Successfully sent database connection request")
```

Comment:
Can we say something like "Successfully configured the connection.  Please run the transponder list command to verify that the connection is active"